common/src/main/java/com/gregtechceu/gtceu/integration/kjs/GregTechKubeJSPlugin.java

```diff
@@ -130,10 +130,13 @@ public void registerBindings(BindingsEvent event) {
         super.registerBindings(event);
         event.add("GTRegistries", GTRegistries.class);
         event.add("GTMaterials", GTMaterials.class);
+        event.add("GCyMMaterials", GCyMMaterials.class);
```

Comment:
GCyMMaterials doesn't have any static fields, only the initializers. This is practically useless to have.